stanford-earth/SE3_BLT
# EARTH-157 Changed stanford_news from SU-SWS to forked stanford-earth …

# READY FOR REVIEW

# Summary
- Fork stanford_news content type from SU-SWS to stanford-earth, add an "Earth Matters Topic" taxonomy vocabulary, and add a taxonomy term reference to the new taxonomy from stanford_news content type. 

# Needed By (Date)
- Next release.

# Urgency
- none

# Steps to Test

1. Change to your 'se3_blt' directory.
2. git fetch; git checkout EARTH-157
3. composer update -prefer-source
4. cd docroot/modules/stanford/stanford_news
5. git remote -v #result should show stanford-earth instead of SU-SWS
6. git fetch; git checkout EARTH-157
7. in browser, go to http://earth.local/admin/config/development/features/edit/stanford_news; import missing
8. in browser, go to http://earth.local/admin/config/development/features/diff/stanford_news; import changes
9. in browser, go to http://earth.local/admin/structure/taxonomy; should see "Earth Matters Topics" vocabulary
10. add term; enter name and pick "Challenge Area" term from "Research Areas" vocabulary
11. in browser, go to http://earth.local/node/add/stanford_news; should be an "Earth Matters Topic" reference field below the "Category" field.

# Affected versions or modules

# Associated Issues and/or People
See associated pull request for EARTH-157 in stanford-earth/stanford_news
# See Also
